challenge1/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
hop hop hop ! C'est pas indenté ça ! :)

Reply:
Ah ! j'avais pas fait attention, le raccourci emmet le met comme ça par défaut mais c'est noté :)